lua/py_lsp/python/strategies.lua

```diff
@@ -58,16 +64,26 @@ M.conda = function(_, venv_name)
 	for _, raw_env in ipairs(raw_env_list) do
 		local env = string.match(raw_env, '[^%s"]+')
 
-        if venv_name == get_last_dir(env) then
-          return path.join(env, "bin", "python")
-        end
+		if venv_name == get_last_dir(env) then
+			return path.join(env, "bin", "python")
+		end
 
 		table.insert(found_envs, path.join(env, "bin", "python"))
 	end
 
 	return found_envs
 end
 
+M.hatch = function(_, venv_name)
+	local venv = vim.fn.trim(vim.fn.system("hatch env find"))
+	print(venv)
```

Comment:
I noticed that I left this print @HallerPatrick...

Reply:
Removed it :)